port: /dev/ttyUSB0
baud: 57600
rate: 20
use_sync: True

dynamixels: {
    head_pan_joint: {id: 1, max_speed: 360, min_angle: -145, max_angle: 145},
    head_tilt_joint: {id: 2, max_speed: 360, min_angle: -90, max_angle: 90},
    right_shoulder_pan_joint: {id: 3, max_speed: 360},
    right_shoulder_lift_joint: {id: 4, max_speed: 360},
    right_arm_roll_joint: {id: 5, max_speed: 360},
    right_elbow_joint: {id: 13, max_speed: 360, invert: True},
    right_wrist_joint: {id: 6, max_speed: 360},
    left_shoulder_pan_joint: {id: 7, max_speed: 360},
    left_shoulder_lift_joint: {id: 8, max_speed: 360},
    left_arm_roll_joint: {id: 9, max_speed: 360},
    left_elbow_joint: {id: 12, max_speed: 360},
    left_wrist_joint: {id: 10, max_speed: 360, invert: True},
    torso_joint: {id: 11, max_speed: 360}
}

controllers: {
    joint_controller: {type: joint_controller, joints: [head_pan_joint, head_tilt_joint, left_shoulder_lift_joint, left_shoulder_pan_joint, left_arm_roll_joint, left_elbow_joint, left_wrist_joint, right_shoulder_lift_joint, right_shoulder_pan_joint, right_arm_roll_joint, right_elbow_joint, right_wrist_joint, torso_joint] }
}
